peekingduck/configs/heuristic/fps.yml

```
@@ -0,0 +1,4 @@
+input: ["pipeline_end"]
+output: ["fps"]
+
+moving_avg: False   # Enables cli logging of moving average FPS per frame
```

```suggestion
fps_log_freq: 100    # Logging frequency of fps in cli
dampen_fps: True
```
fps_log_freq - similar to input.live, log the moving average (not global)
dampen_fps - if true (default), return moving average. If false, return frame_fps. Reason - users may use csv_writer to log fps. If they use moving average, they'll notice a discrepancy with the global result at the end.